server.js

```diff
@@ -39,11 +36,12 @@ class Metadata {
         this.last_updated = new Date();
         this.count = 0;
 
-        if (args.redis) {
+        if (INDEXER === "redis") {
             this.idx = new redisIndexer();
             this.idx.create();
```

Comment:
you can move this outside the if-clause - we can clean that up later tho

Reply:
Did you mean this.idx.create()? See changes.

Reply:
right - why not simply bake this into the custructor for instance - you only need an initialization method if it needs to be called separately from the constructor which isn't the case here.